src/Controller/Cliente/ClienteFormController.php

```diff
@@ -20,47 +21,54 @@ public function __construct(ClienteService $clienteService)
     }
 
     #[Route("/clientes/adicionar", name: "RegistrarCliente", methods: ['POST'])]
-    public function registrarCliente(Request $request,ValidarCpfService $validarCpfService): Response
+    public function registrarCliente(Request $request,ValidarCpfService $validarCpfService, ClienteRepository $clienteRepository): Response
     {
         $token = $request->request->get("_csrf_token");
 
         if (!$this->isCsrfTokenValid("addCliente", $token)) {
             $this->addFlash("danger", "Token CRSF inválido!");
             return $this->redirectToRoute("clientes");
         }
+
+        if(!$validarCpfService->execute($request->request->get("cpf"))){
```

Comment:
assim vc reaproveita

```suggestion
$cpf = $request->request->get("cpf"));
        if(!$validarCpfService->execute(){
```